docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,52 @@
+# Cluster certificate rotation
+
+Instructions on rotating TLS CA and certificates for an AKS Engine cluster.
+
+## Prerequesites
+
+- Kubernetes version 1.8.0 or later is required
+- The etcd members MUST be in a healthy state before rotating the CA and certs.
```

Comment:
We might want to specify more details. I assume you mean that not only is an elected leader actively serving requests, but that all nodes in the cluster are online and available? Or is it only important that they have a working configuration, and not that they literally have to be online?

Reply:
I think the easiest way to meet that requirement is to check that when you run `etcdctl cluster-health` all peers are healthy and cluster is healthy.